### ISSUE TITLE
Option to make Autocomplete keywords configurable between uppercase and lowercase

### ISSUE THREAD

**Issue by KeeonTabrizi** [metabase/metabase#4469]
Right now in the native query editor if I type keywords such as `SELECT` or `FROM` the autocomplete dropdown returns lower cased versions of the keywords:

![image](https://cloud.githubusercontent.com/assets/3408390/23475578/e21f3ebc-fe6c-11e6-8efa-6de09659839b.png)

I think this should either be configurable or default to upper case instead of lower.

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by camsaul** [comment]
Those keywords are part of the ACE Editor's SQL mode autocomplete lib (this isn't a choice we made ourselves). And since SQL keywords are case-insensitive it's really a matter of preference. I could see someone asking for the keywords to be lowercase if they were always uppercase.

**Comment by KeeonTabrizi** [comment]
Fair point. Perhaps I could modify the autocomplete lib, if you have the path on hand that would be great otherwise I can dig for it.

Still, I'd say convention is uppercase and most (if not all) style guides I've seen (highly referenced: http://www.sqlstyle.guide/) use uppercase.

**Comment by camsaul** [comment]
ACE Editor is here https://github.com/ajaxorg/ace

I think maybe sending them a PR would make the most sense

**Comment by fulldecent** [comment]
@KeeonTabrizi That guide is an excellent reference. Do you know of any library that enforces (autocorrects) SQL into the river format?

**Comment by stroupaloop** [comment]
DBeaver is a great example of configurability here that I'd love to include. I prefer all CAPS when it comes to keywords for visual distinction.

**Comment by Tony-metabase** [comment]
Recently we Replace ACE with CodeMirror in the native query editor so this became the opposite ... Will rewrite this feature request to make it apply to recent changes.

<img width="613" height="290" alt="Image" src="https://github.com/user-attachments/assets/45462f17-adf6-451b-a64d-f57573c6acfe" />
